src/storage/disk_manager.c

```diff
@@ -1205,24 +1207,57 @@ disk_get_link (THREAD_ENTRY * thread_p, INT16 volid, INT16 * next_volid, char *n
   return next_volext_fullname;
 }
 
+/*
+ * disk_adapt_other_config_volume_full_name_to_local_config - adapt the full volume path and name
+ *                      coming, possibly, from an old or different config to the current config
+ */
+void
+disk_adapt_other_config_volume_full_name_to_local_config (char *local_config_full_name,
```

Comment:
Make the name a bit shorter by dropping "other_config". It is implied and make the function easier to read/use.
```suggestion
disk_adapt_volume_full_name_to_local_config (char *local_config_full_name,
```

Reply:
Done